pubsub/aws/snssqs/metadata.yaml

```diff
@@ -0,0 +1,160 @@
+# yaml-language-server: $schema=../../../component-metadata-schema.json
+schemaVersion: v1
+type: pubsub
+name: aws.snssqs
+version: v1
+status: stable
+title: "AWS SNS/SQS"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-aws-snssqs/
+capabilities:
+  - ttl
+builtinAuthenticationProfiles:
+  - name: "aws"
+metadata:
+  - name: region
+    required: true
+    description: |
+      The AWS region where the SNS/SQS assets are located or be created in. See the regions product services page.
+      Ensure that SNS and SQS are available in that region.
+    url:
```

Comment:
Is this correct? cc @ItalyPaleAle

Reply:
I think so although a nit would be to make the title "Regions Product Services" a bit more descriptive

Reply:
@ItalyPaleAle could you please clarify the ask here?

Reply:
```
title: "Supported AWS services per region"
```